cartofacade/capi.go

```diff
@@ -239,15 +239,14 @@ func (vc *Carto) getPointCloudMap() ([]byte, error) {
 		return nil, err
 	}
 
+	pcd := bstringToByteSlice(value.point_cloud_pcd)
```

Comment:
needed to move this as otherwise `value.point_cloud_pcd` is freed & zeroed out before it is converted into a go slice, resulting in a panic.